pyannote/audio/core/io.py

```diff
@@ -317,6 +330,9 @@ def crop(
             sample_rate = info.sample_rate
             frames = info.num_frames
 
+            if callable(getattr(file["audio"], "read", None)):
+                file["audio"].seek(0)
```

Comment:
See https://github.com/pytorch/audio/issues/1442

Reply:
Yes this is the problem that I was avoiding because torchaudio info doesn't seek back. Nice fix for it. I'll review the rest of the comments about this soon.

Reply:
Or maybe we should drop support for file-like object in Audio.crop, and only support it in Audio.__call__